### PR TITLE
Added Script to check Lua Abilites and Items

### PR DESCRIPTION
**Pull request by Chronophylos**
Added `.lua` to all "ScriptFile" Keys
Fixed AbilitySpecial Numbering of Charger's Charge and Shroud 1
Fixed Item Turret and Spell Totem not not crash parseKV cause of an empty value
FIxed Satanic Core looking for a non existing Special Value "radius"
Removed leftover variables in trumps_fists.lua
Removed unused variable in reactive_reflex.lua
Removed leftover pull staff file
Removed not working write-version.js

### CODE REVIEW

File: game/scripts/vscripts/items/satanic_core.lua [hunk 1]
```diff
@@ -66,7 +64,7 @@ function modifier_item_satanic_core:GetAuraSearchFlags()
 end
 
 function modifier_item_satanic_core:GetAuraRadius()
-  return self.aura_radius
+  return 0
```
Comment (chrisinajar) [inline game/scripts/vscripts/items/satanic_core.lua]: Doesn't this power the spell steal?

**Comment by chrisinajar** [comment]
Risky for 1.0, seems to be all stylistic as opposed to functional, going to wait for now

File: game/scripts/vscripts/items/satanic_core.lua (hunk 1 above)
Comment (Chronophylos) [inline game/scripts/vscripts/items/satanic_core.lua]: I don't know. I removed it because the Special Value 'radius' didn't exist

**Comment by Chronophylos** [comment]
You talked about this and I wanted to do it.

**Comment by chrisinajar** [comment]
Hahaha, it's awesome work, it's just hard to review moments before 1.0 release 💤 💀

**Comment by Chronophylos** [comment]
At least I found Bugs with charger and shroud

**Comment by chrisinajar** [comment]
FUCK IT WE'RE DOING IT LIVE

**Comment by Chronophylos** [comment]
ok.
your call.

**Pull request merged**